src/main/cpp/processors/flanger.cpp

```diff
@@ -209,7 +233,33 @@ void Flanger::process( AudioBuffer* sampleBuffer, bool isMonoSource )
 void Flanger::setSweep()
 {
     // translate sweep rate to samples per second
-    _step = ( SAMPLE_TYPE ) ( _sweepSamples * 2.0 * _sweepRate ) / ( SAMPLE_TYPE ) AudioEngineProps::SAMPLE_RATE;
+    _step = ( _sweepSamples * 2.0 * _sweepRate ) / ( SAMPLE_TYPE ) AudioEngineProps::SAMPLE_RATE;
     _maxSweepSamples = _sweepSamples;
     _sweep = 0.0;
 }
+
+void Flanger::init( float rate, float width, float feedback, float delay, float mix )
+{
+    FLANGER_BUFFER_SIZE = ( int ) (( SAMPLE_TYPE ) AudioEngineProps::SAMPLE_RATE / 5.0f );
+    SAMPLE_MULTIPLIER   = ( SAMPLE_TYPE ) AudioEngineProps::SAMPLE_RATE * 0.01f;
+
+    _writePointer    = 0;
+    _feedbackPhase   = 1.f;
+    _sweepSamples    = 0.f;
```

Comment:
Feedback phase was always defined as 0, making the feedback actually... not work.